.github/workflows/release.yml

```diff
@@ -199,4 +199,6 @@ jobs:
         CHANGELOG_FILE: "CHANGELOG.md"
         ALLOW_EMPTY_CHANGELOG: "true"
       with:
-        args: components/**/build/libs/*.jar
+        args: |
+          components/**/build/libs/*.jar
+          components/**/**/build/libs/*.jar
```

Comment:
Shouldn't the first glob pattern also match the 2nd because of using `**` ?

Reply:
that was my assumption as well, but it currently does not. [here is the _spec_](https://www.gnu.org/software/bash/manual/html_node/Pattern-Matching.html)
Here is the result without that change https://github.com/microsoft/kiota-java/releases/tag/v1.3.0

Here is a local test

```bash
./kiota-java$ ls components/**/build/libs/*.jar
components/abstractions/build/libs/abstractions-javadoc.jar  components/abstractions/build/libs/abstractions.jar  components/bundle/build/libs/bundle-sources.jar
components/abstractions/build/libs/abstractions-sources.jar  components/bundle/build/libs/bundle-javadoc.jar      components/bundle/build/libs/bundle.jar
./kiota-java$ ls components/**/**/build/libs/*.jar
components/authentication/azure/build/libs/azure-javadoc.jar  components/serialization/json/build/libs/json-javadoc.jar
components/authentication/azure/build/libs/azure-sources.jar  components/serialization/json/build/libs/json-sources.jar
components/authentication/azure/build/libs/azure.jar          components/serialization/json/build/libs/json.jar
components/http/okHttp/build/libs/okHttp-javadoc.jar          components/serialization/multipart/build/libs/multipart-javadoc.jar
components/http/okHttp/build/libs/okHttp-sources.jar          components/serialization/multipart/build/libs/multipart-sources.jar
components/http/okHttp/build/libs/okHttp.jar                  components/serialization/multipart/build/libs/multipart.jar
components/serialization/form/build/libs/form-javadoc.jar     components/serialization/text/build/libs/text-javadoc.jar
components/serialization/form/build/libs/form-sources.jar     components/serialization/text/build/libs/text-sources.jar
components/serialization/form/build/libs/form.jar             components/serialization/text/build/libs/text.jar
```